EnglishTranslation-master/ETC.tsv

```diff
@@ -19986,27 +19986,27 @@ ETC_20151224_019986	Failed the test
 ETC_20151224_019987	Open the portal to go the den of monsters to purify monsters by attracting them to magic circles
 ETC_20151224_019988	The mine wagon will automatically forward to the front by pushing it. Lead the Mine Wagon to the safe place
 ETC_20151224_019989	The floor trap and rotation point are activated by pressing the switch.{nl}Additionally, you can destroy the stone.
-ETC_20151224_019990	The wagon will be destroyed if drop to caved area.{nl}Be cautious
+ETC_20151224_019990	The wagon will be destroyed if drop to caved area.{nl}Be cautious.
 ETC_20151224_019991	The wagon has been destroyed!{nl}Push the wagon to create another one.
 ETC_20151224_019992	The game has begun.
-ETC_20151224_019993	You will return to the zone after 1 second
-ETC_20151224_019994	Failed the event{nl}You will return to the starting point
+ETC_20151224_019993	You will return to the zone after 1 second.
+ETC_20151224_019994	Failed the event.{nl}You will return to the starting point.
```

Comment:
No periods (it'll look nicer that way since this appears on the center of your screen)

(applies till ETC_20151224_020003)